bin/test-runner/w3f/accumulate.ts

```diff
@@ -92,8 +97,9 @@ class TestState {
       ),
       privilegedServices: PrivilegedServices.create({
         manager: privileges.bless,
-        authManager: tryAsPerCore(privileges.assign, chainSpec),
-        validatorsManager: privileges.designate,
+        assigners: tryAsPerCore(privileges.assign, chainSpec),
+        delegator: privileges.designate,
+        registrar: privileges.register ?? tryAsServiceId(0),
```

Comment:
since `0` might be a regular service, I'd rather suggest using some large value for the placeholder:
```suggestion
        registrar: privileges.register ?? tryAsServiceId(2**32 - 1),
```